client/src/main/scala/sbtcompendium/models/ErrorResponse.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2019-2020 47 Degrees, LLC. <http://www.47deg.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package sbtcompendium.models
+
+import io.circe._
+import io.circe.generic.semiauto._
+
+final case class ErrorResponse(message: String)
```

Comment:
Where are we using this ErrorResponse case class?

Reply:
https://github.com/higherkindness/sbt-compendium/blob/9a5da27a5415ca60b3e01b32317b6861c36c2267/client/src/main/scala/sbtcompendium/client/CompendiumClient.scala#L135

Reply:
Ah, I see... Okay, thanks! 🙂